Index.html

```diff
@@ -0,0 +1,153 @@
+    <title>Bokolo Drill</title>
+</head>
+<body>
+  <div class="center">
+    <span></span><span></span><span></span>
+  </div>
+    <header class="header-container">
+      <div class="circle"></div>
+      <nav>
+        <ul class="first-navbar">
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li><a>English</a></li>
+          <li><a>My account</a></li>
+          <li><a>logout</a></li>
+        </ul>
+      </nav>
+        <nav class="navbar">
+          <a href="Index.html"><img class="brand-logo" src="Images/logo_image/logo-image.png" alt="Brand-name-and-logo"></a>
+         <ul class="nav-list-items">
+          <li class="list-item"><a  href="Index.html">Home</a></li>
+          <li class="list-item"><a  href="Interfaces/About.html">About</a></li>
+          <li class="list-item"><a  href="#">Program</a></li>
+          <li class="list-item"><a  href="#">Sponsor</a></li>
+          <li class="list-item"><a  href="#">New</a></li>
+          <li class="list-item"><a  href="#">Ticket</a></li>
+         </ul>
+         <div class="hambuger-container">
+         <div class="bar"></div>
+         <div class="bar"></div>
+         <div class="bar"></div>
+         </div>
+        </nav>
+      </header>
+      <section class="landing-section">
+        <div class="intro-section">
+          <h1 class="openingText">
+            Quarter mboko rappers           
+          </h1>
+          
+               <h2 class="captureText">
+                Presenting! Mboko gods among us 2022
+               </h2>
+          
+
+               <p class="explaining-text">
+                Come one come all. Introducing the best covered up mboko divine beings and goddesses among us this year in December. Get ready to be stunned by these incredible rappers.
+               </p>
+
+               <p class="eventDate">2022.12.10(SAT) ~ 11(SUN)</p>
+               <span class="venue">At Congrate hall Bamenda Cameroon</span>
+        </div>
+      </section>
+
+      <section class="shedule-section">
+        <div class="text-time">
+          <h2 class="text-header">Main Program</h2>
+          <hr class="line">
+        </div>
+        
+        <div class="shedule-container">
+          <div class="shedule-cards">
+            <img class="image" src="Images/Musician/dj.png"  alt=" ">
+            <h3>Openning with JD</h3>
+            <p class="text1">Listen to theamazing talented DJs in Bemanda. Give you what they do best in their career</p>
+          </div>
+
+          <div class="shedule-cards">
+            <img class="image" src="Images/Musician/artist.png" alt=" ">
+            <h3>1<sup>st</sup> Artist</h3>
+            <p>The fist artist will take over the stage after the Djs have finish performing their talent</p>
+          </div>
+
+          <div class="shedule-cards">
+            <img class="image" src="Images/Musician/artist.png" alt=" ">
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image